core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -554,6 +555,7 @@ public GraphHopper init(CmdArgs args) {
 
         // elevation
         String eleProviderStr = args.get("graph.elevation.provider", "noop").toLowerCase();
+        this.smoothElevation = args.getBool("graph.elevation.smooth_elevation", false);
```

Comment:
also I'd slightly prefer to avoid "elevation" wording duplication and instead use `graph.elevation.smoothing`?

Reply:
Yes makes sense, I will change this :).